content/Helix-Guidelines.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Helix Guidelines # this is the Title of your Blip
+ring: Adopt # Values: [Hold | Assess | Trial | Adopt]
+quadrant: Techniques # Values: [Tools-And-Libraries | Techniques | Modules | Products] - Make sure to keep these exact values, the Radar is also case sensitive.
+isNew: TRUE # Values: [TRUE | FALSE] - Make sure to keep the value all uppercase.
+---
+
+The [Helix Guidelines](https://helix.sitecore.com/) should not require an introduction anymore. It is the defacto standard for Sitecore developers to structure, architect and build Solutions.
```

Comment:
`defacto` seems to be a company :) According to [Wikipedia](https://en.wikipedia.org/wiki/De_facto), `de facto` would be correct

Reply:
Latin is hard :) - thanks for your thorough review!